### ISSUE TITLE
Hubot converting '&' to '&amp;' - breaks some image links in Hipchat web client

### ISSUE THREAD

**Issue by JustinMullin** [thenerdery/nerdbot#54]
When hubot prints a message, ampersands are URL encoded, which is improper for Hipchat, and breaks some image links ("Failed to load the image").  To reproduce, try 'nerdbot cast storm crow' in the Hipchat web client.  The resulting image response from Nerdbot should fail to load.

The native client seems to load these URLs successfully, I only see this issue in the web client.  I assume this is related to https://github.com/hipchat/hubot-hipchat/pull/247 and can be addressed by updating the hubot module.

**Comment by JustinMullin** [comment]
Hey Nerdbot maintainers - I believe the fix for this is just an npm update.  Any chance someone with access can try that?  It would be a quick usability improvement for those of us on the Hipchat web client.  =)  Thanks!

**Comment by jcarlso5** [comment]
A node update throws up all over itself. I don't have time to dig into the error, but if you know how to resolve it, please let me know:
```
In file included from ../src/binding.cc:18:
../node_modules/nan/nan.h:318: error: 'REPLACE_INVALID_UTF8' is not a member of 'v8::String'
```

**Comment by JustinMullin** [comment]
Hmm, okay.  I'll see if I can reproduce, but update worked fine for me.  Thanks for trying @jcarlso5, much appreciated.

**Comment by kensykora** [comment]
What version of node you guys running @jcarlso5

**Comment by jcarlso5** [comment]
I was able to update node to 0.10.40 and update all nerdbot packages. Please test!

**Comment by kensykora** [comment]
Fixed!

![screen shot 2015-09-11 at 10 38 40 am](https://cloud.githubusercontent.com/assets/64756/9819048/4ebdc1b8-5871-11e5-8a51-680c1145029b.png)

**Comment by kensykora** [comment]
kinda? I mean the image shows up but the URL still seems weird.

**Comment by jcarlso5** [comment]
Appears to be broken in Chrome 45 on Mac as well
![screenshot 2015-09-11 10 57 46](https://cloud.githubusercontent.com/assets/7442961/9819493/f8823d62-5873-11e5-94b7-03be3aefd018.png)

**Comment by JustinMullin** [comment]
Yeah, it's only an issue from the web client.  The problem is the &amp; encoding in the URL, which the web client doesn't like - looks like it's still encoding the ampersand so the fix didn't work.  Last I checked it was working properly from a clean checkout and build, so I'm not sure what else might be wrong.  I'll take a second look next time I have a chance.

**Comment by Bendetta** [comment]
I have this problem with the desktop apps for both Mac (El Capitan) and Windows (8.1)

**Comment by kensykora** [comment]
Can we try updating modules again and see if this fixes the issue? Otherwise we'll need to dig deeper.
